app/views/shelters/show.html.erb

```diff
@@ -10,8 +10,12 @@
 
 <%= link_to 'Add Review', "/shelters/#{@shelter.id}/reviews/new" %>
 <% @shelter.reviews.each do |review| %>
+<section id="review<%= review.id %>"
```

Comment:
added the section id to be able to test for specific sections using `within`

Reply:
Wonderful. I think this will look good for evals.